### PR TITLE
Digish

### LINKED ISSUE #12: Get a function similar to miekg/exdns/q to work
miekg/exdns/q is a command line tool in go that simulates dig, we should try to get this to work with revel.
https://github.com/miekg/exdns/tree/master/q

To start it should just accept a zone and use static configured flags.

### LINKED ISSUE #17: tag DNS Response Message for frontend consumption

### PR DESCRIPTION
**Pull request by romu42**
Started working on how we send queries, still need to add tests. I also stared looking at how we can tag parts of the message headers and flags.

### CODE REVIEW

**Comment by romu42** [comment]
This is not ready for merge as of yet.

File: app/controllers/app.go [hunk 1]
```diff
@@ -49,3 +76,26 @@ func (c App) Index() revel.Result {
 	}
 	return c.Render(Zone, response, rdata)
 }
+
+func init() {
```
Comment (lundqvistm) [inline app/controllers/app.go]: Not a big fan of the init() functions and I think revel has this function `https://pkg.go.dev/github.com/revel/revel#OnAppStart` which could be used instead, wdyt?

File: app/controllers/app.go [hunk 2]
```diff
@@ -3,34 +3,61 @@ package controllers
...
+	//nameserver = "8.8.8.8"
 
-	client.Net = "tcp"
+	nameserver = "10.30.7.4"
+	rd = true
 
-	response, _, err := client.Exchange(m, "8.8.8.8:53")
-	rdata := dns.Field(response.Answer[0], 1)
+	// build message by sections
+	//message := createMessage(rd)
```
Comment (lundqvistm) [inline app/controllers/app.go]: Remove the code that is commented out.

**Review by lundqvistm — COMMENTED** [review]
I get a panic "dial tcp 10.30.7.4:53: i/o timeout" when I try to get information for a zone so I can't really try it out :(

File: app/controllers/app.go (hunk 2 above)
Comment (romu42) [reply app/controllers/app.go]: Fixed.

File: app/controllers/app.go (hunk 1 above)
Comment (romu42) [reply app/controllers/app.go]: I don't see how to use it to register the Template functions. Initial test doesn't seem to work, I probably need another pair of eyes on it.

Comment (romu42) [reply app/controllers/app.go]: https://revel.github.io/manual/templates-go.html#CustomFunctions

The example shows the function being registered using a init() function.
```
func init() {
    revel.TemplateFuncs["my_eq"] = func(a, b interface{}) bool {
        return a == 0  || a == b
    }
}
```

Comment (romu42) [reply app/controllers/app.go]: Works now as suggested by @lundqvistm .

**Review by lundqvistm — APPROVED** [review]

**Pull request merged**